default_app/renderer.ts

```diff
@@ -27,8 +27,8 @@ function initialize () {
   }
 
   document.querySelector<HTMLAnchorElement>('.electron-version')!.innerText = `Electron v${process.versions.electron}`
-  document.querySelector<HTMLAnchorElement>('.chrome-version')!.innerText = `Chromium v${process.versions.chrome}`
-  document.querySelector<HTMLAnchorElement>('.node-version')!.innerText = `Node v${process.versions.node}`
+  document.querySelector<HTMLAnchorElement>('.chrome-version')!.innerText =`<a href="https://www.chromium.org/v${process.versions.chrome}">Chromium v${process.versions.chrome}</a>``
```

Comment:
This commit probably shouldn't be in this PR?

Reply:
Ya my mistake thanks for pointing that out